### DISCUSSION TITLE
Minimal Squelch level.

### DISCUSSION THREAD

**Discussion by slavikby** [egzumer/uv-k5-firmware-custom#382] [Q&A] [answered]
Hello. Please tell me - how can I make squelch more sensitive?

**Comment by ErikS-web** [comment]
Go2 spectrumscanner, There you can adjust with 4 parameters.
See under [Spectrum-analyzer#detail-monitor-screen](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Spectrum-analyzer#detail-monitor-screen)

**Comment by egzumer** [comment, marked as answer]
you can edit squelch with the CHIRP driver now https://github.com/egzumer/uvk5-chirp-driver/releases/tag/v0.8
